testbed/core/templates/oauth2_provider/authorize.html

```diff
@@ -0,0 +1,54 @@
+{% extends "_base.html" %}
+
+{% block content %}
+<main class="oauth-authorize">
+  <div class="authorization-container">
+    <h1>Account Data Transfer Request</h1>
+    
+    <div class="authorization-info">
+      <p class="lead">
+        <strong>{{ application.name }}</strong> is requesting permission to transfer your ActivityPub account data.
```

Comment:
We should definitely keep this for local-only roundtrip testing as well as we're going to need it when the testbed acts as a resource server.